pkg/osquery/runtime/runner.go

```diff
@@ -377,6 +377,9 @@ func (r *Runner) launchOsqueryInstance() error {
 	}
 
 	span.AddEvent("socket_created")
+	r.slogger.Log(ctx, slog.LevelDebug,
```

Comment:
I wonder if it would be interesting to collect the spans from testing -- it looks like they already have this.

Reply:
Maybe! I imagine it could be interesting if we wanted to do benchmarking in the future too